releasenotes/notes/prelude-release-7.40.0-0ea3b50acc7e2b1c.yaml

```
@@ -0,0 +1,5 @@
+prelude:
+    |
+    Release on: 2022-10-28
+
+    - Please refer to the `7.40.0 tag on integrations-core <https://github.com/DataDog/integrations-core/blob/master/AGENT_CHANGELOG.md#datadog-agent-version-7400>`_ for the list of changes on the Core Checks
```

```suggestion
    - Please refer to the `7.40.0 tag on integrations-core <https://github.com/DataDog/integrations-core/blob/master/AGENT_CHANGELOG.md#datadog-agent-version-7400>` for the list of changes on the Core Checks
```
I'm guessing this underscore is a typo

---

I don't think so, it's actually a named link in restructured text

---

https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#external-links